nipoppy/workflow/bids_conv/run_dcm2bids.py

```diff
@@ -0,0 +1,233 @@
+    
+        if stage == 2:
+            for fpath in fpaths_to_copy:
+                fpath = Path(fpath)
+                logger.info(f"Copying {fpath} to {DATASET_ROOT}/proc/{fpath.name} (to be seen by Singularity container)")
+                shutil.copyfile(fpath, f"{DATASET_ROOT}/proc/{fpath.name}")
+
+        if n_jobs > 1:
+            ## Process in parallel! (Won't write to logs)
+            dcm2bids_results = Parallel(n_jobs=n_jobs)(delayed(run_dcm2bids)(
+                dicom_id, global_configs, session_id, stage, overlays, logger
+                ) for dicom_id in dcm2bids_participants)
+
+        else:
+            # Useful for debugging
+            dcm2bids_results = []
+            for dicom_id in dcm2bids_participants:
+                res = run_dcm2bids(dicom_id, global_configs, session_id, stage, overlays, logger) 
+            dcm2bids_results.append(res)
+
+        # Check successful dcm2bids runs
+        n_dcm2bids_success = np.sum(dcm2bids_results)
+        logger.info(f"Successfully ran dcm2bids (Stage 1 or Stage 2) for {n_dcm2bids_success} out of {n_dcm2bids_participants} participants")
+
+        # Check successful bids (NOTE: will count partial conversion as successful)
+        participants_with_bids = {
+            parse_file_entities(dpath)['subject']
+            for dpath in
+            glob.glob(f"{bids_dir}/sub-*/{session}")
+        }
+
+        new_participants_with_bids = dcm2bids_participants & participants_with_bids
+        
+        logger.info("-"*50)
+
+        if stage == 1:
+            # Generate empty config file
+            logger.info(f"Creating dcm2bids_config.json in {DATASET_ROOT}/proc (to be seen by Singularity container)")
+            
+            config_content = {
+                "descriptions": []
+            }
+
+            with open(f"{DATASET_ROOT}/proc/dcm2bids_config.json", 'w') as config_file:
+                json.dump(config_content, config_file, indent='\t')
+
+            logger.info("Stage 1 done! Still need to run Stage 2")
+
+        else:
+
+            logger.info(f"Current successfully converted BIDS participants for session {session}: {len(participants_with_bids)}")
+            logger.info(f"BIDS conversion completed for the {len(new_participants_with_bids)} out of {len(dcm2bids_participants)} new participants")
+            
+            if len(new_participants_with_bids) > 0:
+                dcm2bids_df.loc[dcm2bids_df[COL_DICOM_ID].isin(new_participants_with_bids), COL_CONV_STATUS] = True
+                df_doughnut.loc[dcm2bids_df.index] = dcm2bids_df
+                save_backup(df_doughnut, fpath_doughnut, DNAME_BACKUPS_DOUGHNUT)
+
+    else:
+        logger.info(f"No new participants found for bids conversion...")
+
+    logger.info("-"*50)
+    logger.info("")
+
+if __name__ == '__main__':
+    # argparse
+    HELPTEXT = """
+    Script to perform DICOM to BIDS conversion using dcm2bids
+    """
```

Comment:
Could you add somewhere (either here or in the help message for `--stage`) that, if running Stage 2, the config file is expected to be found at `<DATASET_ROOT>/proc/dcm2bids_config.json`?